Signal/src/ViewControllers/ConversationView/Cells/MediaGalleryCellView.swift

```diff
@@ -142,11 +146,39 @@ public class MediaGalleryCellView: UIStackView {
                 moreLabel.autoCenterInSuperview()
             }
         }
+
+        for itemView in itemViews {
+            guard !moreItemViews.contains(itemView) else {
+                // Don't display the caption indicator on
+                // the "more" item, if any.
+                continue
+            }
+            guard let index = itemViews.index(of: itemView) else {
+                owsFailDebug("Couldn't determine index of item view.")
+                continue
+            }
+            let item = items[index]
+            guard let caption = item.caption else {
+                continue
+            }
+            guard caption.count > 0 else {
+                continue
+            }
+            guard let icon = UIImage(named: "media_album_caption") else {
+                owsFailDebug("Couldn't load icon.")
+                continue
+            }
+            let iconView = UIImageView(image: icon.withRenderingMode(.alwaysTemplate))
+            iconView.tintColor = .ows_white
```

Comment:
How does this look over a white image? I wonder if it'd be better to have an asset with a built in shadow, but that might be more a question for @myles-signal

Reply:
I was going to ask Myles the same thing.  I've added it to the media album Qs card so you can track his answer.